pkg/test/e2e/utils/client.go

```diff
@@ -449,6 +449,9 @@ func (r *TestClient) CreateAWSCluster(projectID, dc, name, secretAccessKey, acce
 				},
 			},
 			Version: models.Semver(version),
+			ClusterNetwork: &models.ClusterNetworkingConfig{
+				KonnectivityEnabled: true,
+			},
```

Comment:
Won't this enable konnectivity for all e2e tests using `CreateAWSCluster`? Do we really want that?

Reply:
Good catch. I meant to remove it. Fixing...